checks/mysql

```diff
@@ -250,3 +248,171 @@ check_info['mysql.connections'] = {
+#   |       | |  _ / _` | |/ _ \ '__/ _` | \___ \| | | | '_ \ / __|        |
+#   |       | |_| | (_| | |  __/ | | (_| |  ___) | |_| | | | | (__         |
+#   |        \____|\__,_|_|\___|_|  \__,_| |____/ \__, |_| |_|\___|        |
+#   |                                             |___/                    |
+#   |                    ____  _        _                                  |
+#   |                   / ___|| |_ __ _| |_ _   _ ___                      |
+#   |                   \___ \| __/ _` | __| | | / __|                     |
+#   |                    ___) | || (_| | |_| |_| \__ \                     |
+#   |                   |____/ \__\__,_|\__|\__,_|___/                     |
+#   |                                                                      |
+#   +----------------------------------------------------------------------+
+
+def inventory_mysql_galerasync(parsed):
+    for instance, values in parsed.iteritems():
+        if values.get('wsrep_provider', 'none') != 'none' and u'wsrep_local_state_comment' in values:
+            yield instance, {}
+
+@get_parsed_item_data
+def check_mysql_galerasync(item, _no_params, values):
+    if u'wsrep_local_state_comment' in values:
+        if values[u'wsrep_local_state_comment'] == 'Synced':
+            state = 0
+        else:
+            state = 2
+        return (state, 'WSREP local state comment: %s' % values[u'wsrep_local_state_comment'])
+
+check_info['mysql.galerasync'] = {
+    "inventory_function"      : inventory_mysql_galerasync,
+    "check_function"          : check_mysql_galerasync,
+    "service_description"     : "MySQL Galera Sync %s",
+    "has_perfdata"            : False,
+}
+
+#.
+#   .--Galera Donor--------------------------------------------------------.
+#   |      ____       _                  ____                              |
+#   |     / ___| __ _| | ___ _ __ __ _  |  _ \  ___  _ __   ___  _ __      |
+#   |    | |  _ / _` | |/ _ \ '__/ _` | | | | |/ _ \| '_ \ / _ \| '__|     |
+#   |    | |_| | (_| | |  __/ | | (_| | | |_| | (_) | | | | (_) | |        |
+#   |     \____|\__,_|_|\___|_|  \__,_| |____/ \___/|_| |_|\___/|_|        |
+#   |                                                                      |
+#   +----------------------------------------------------------------------+
+
+def inventory_mysql_galeradonor(parsed):
+    for instance, values in parsed.iteritems():
+        if values.get('wsrep_provider', 'none') != 'none' and u'wsrep_sst_donor' in values:
+            yield instance, {'wsrep_sst_donor': values['wsrep_sst_donor']}
+
+@get_parsed_item_data
+def check_mysql_galeradonor(item, params, values):
+    if u'wsrep_sst_donor' in values:
+        if values[u'wsrep_sst_donor'] == params['wsrep_sst_donor']:
+            return (0, 'WSREP sst donor contains "%s"' % values[u'wsrep_sst_donor'])
+        else:
+            return (1, 'WSREP sst donor contains "%s" and not "%s"' % (values[u'wsrep_sst_donor'], params['wsrep_sst_donor']))
```

Comment:
levels/params should be placed in brackets as other checks do:
`
if state:
    infotext += " (was at discovery %s)"
`

Reply:
see commit d9300d4e9c92d52f719e171f7c8f33fb02bc049d